org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/SonarLintUiPlugin.java

```diff
@@ -274,13 +262,9 @@ public static IEditorPart findActiveEditor() {
     return page.getActiveEditor();
   }
 
-  public static void analyzeCurrentFile() {
+  public static void analyzeOpenedFiles() {
     // SLE-122 Delay a little bit to let the time to the workspace to initialize (and avoid NPE)
-    new AnalyzeCurrentFileJob().schedule(2000);
-  }
-
-  public static void addSonarLintPartListener() {
-    new RegisterSonarLintPartListenerJob().schedule();
+    new AnalyzeOpenedFilesJob().schedule(2000);
```

Comment:
Everything is lazy, so I'm not sure it means anything "workspace ready". In any case, I don't know any event that is triggered when all editors are restored.